src/netbase.h

```diff
@@ -247,4 +247,13 @@ void InterruptSocks5(bool interrupt);
  */
 bool Socks5(const std::string& strDest, uint16_t port, const ProxyCredentials* auth, const Sock& socket);
 
+/**
+ * Determine if a port is "bad" from the perspective of attempting to connect
+ * to a node on that port.
+ * @see doc/p2p-bad-ports.md
+ * @param[in] port Port to check.
+ * @return true if the port is bad
```

Comment:
nit, returns both true and false
```suggestion
 * @returns whether the port is bad
```

Reply:
Done.